src/NzbDrone.Core/Configuration/IConfigService.cs

```diff
@@ -68,6 +68,7 @@ public interface IConfigService
         //UI
         int FirstDayOfWeek { get; set; }
         string CalendarWeekColumnHeader { get; set; }
+        int MovieRuntime { get; set; }
```

Comment:
Make this an enum for better readability when called in places, and easier changes.